common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -136,8 +154,6 @@ function (HTML5Video, Resizer) {
         // http://code.google.com/p/gdata-issues/issues/detail?id=4654
         // When the YouTube team fixes the API bug, we can remove this
         // temporary bug fix.
```

Comment:
@smarnach Minor nitpick: It might make sense to move this comment closer to where browserIsFirefox is used (i.e. inside `setPlaybackRate`), as now it's "applied" to a block of code that has nothing to do with this comment. Stale misleading comments are worse than no comments at all